hepdata_validator/data_file_validator.py

```diff
@@ -121,6 +121,9 @@ def validate(self, **kwargs):
                 json_validate(data, custom_schema)
             else:
                 json_validate(data, default_data_schema)
+                if self.schema_version != '0.1.0':
```

Comment:
I realise that in practice at the moment, the schema version will only be 0.1.0 or 1.0.0, but I wonder if we could make this less hard-coded. e.g. check the first part of the version number as an integer.